config/database.php

```diff
@@ -66,9 +66,12 @@
             'strict' => true,
             'engine' => null,
             'sslmode' => env('DB_SSLMODE', 'disabled'),
-            'options' => extension_loaded('pdo_mysql') ? array_filter([
+            'options' => extension_loaded('pdo_mysql') ? [
+                PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => false,
```

Comment:
we do not want to force this.

Reply:
Apologies, this took some effort to fix. I believe from y testing the latest commit will allows people to use SSL/TLS connections or not. I have tested this on my local development system where I do **not** SSL/TLS database end; and on a system where I **do** have SSL/TLS database backend.

SSL/TLS CA, CERT, and KEY are optional. If they are not provided the function `array_filter()` will remove them.

MYSQL_ATTR_SSL_VERIFY_SERVER_CERT is outside `array_filter()` because the function will remove the option if it is falsey; however there are times when that variable/setting must be set to false.